tests/common/dualtor/dual_tor_utils.py

```diff
@@ -455,3 +456,35 @@ def shutdown(vm_names=None, upper=False, lower=False):
     logger.info('Recover T1 VM ports connected to tor')
     for eos_host, vm_intf in down_intfs:
         eos_host.no_shutdown(vm_intf)
+
+
+@pytest.fixture
+def set_upper_tor_active(upper_tor_host):
+    """
+    A function level fixture to toggle mux to upper tor for all interfaces
+    """
+    force_active_tor(upper_tor_host, "all")
```

Comment:
Please use the mux simulator to change the active ToR. Using the CLI commands should be reserved for the test cases that require manually changing the active ToR. Since the CLI commands are a feature that is being tested, we don't want to use them elsewhere.

Reply:
I agree, we should use MUX simulator for toggle usecases. But, what is `force_active_tor` helper to be used for?
We should also make the fixtures in MUX simulator as function scope instead of module.

Reply:
This also makes me ask why do we need this fixtures when we have similar task being done by fixtures present in `mux_simulator_control.py`?
`toggle_all_simulator_ports_to_lower_tor()`
`toggle_all_simulator_ports_to_upper_tor()`
.....

Reply:
I think using CLI is better than using the ```mux_simulator``` interfaces. Actually, the CLI will call  ```mux_simulator``` at last. These fixtures are intended to be used before dualtor test cases that require a certain ToR as active. 
In other word, I don't think we should call ```mux_simulator``` interfaces directly. Test cases should do what human do. So I think CLI is a better choice.